frontend/src/components/eventCreatorTooltip/EventCreatorTooltip.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState } from 'react'
+import './EventCreatorTooltip.css'
+
+export default function EventCreatorTooltip({ children, ...rest }) {
+    const [show, setShow] = useState(false)
+
+    return (
+        <div>
+            <div
+                className="tooltip"
+                style={show ? { visibility: 'visible' } : {}}
+            >
+                Are you sure?
```

Comment:
So the interaction is that when user hovers on top of the "Unregister" button it would display "Are you sure?". Visually it looks quite confusing, with the "Are you sure?" almost as big as the "Unregister" button, with no arrow pointing. Also I am not sure if that is the best user flow, normally we ask for the user confirmation when user clicks the button. Tooltip is generally used to explain a certain UI component, but this one is more like asking additional questions.

Reply:
Thank you. I'll think for another use of the tooltip, I still don't know were to put it (that is visually appealing)

Reply:
To Do

Reply:
echo on this. The "are you sure" also looks like an button -- may need to rethink how you want to present it